mentors/elm/c.json

```
@@ -6,5 +6,13 @@
     "link_url": "",
     "avatar_url": null,
     "bio": "I am drawn to Elm because it is a safe, reliable, and refactorable alternative to building webapps. It's an approachable language that not only supports the dynamic nature of greenfield projects, but it can also be adopted in small steps within legacy apps implemented in other front-end solutions."
+  },
+  {
+    "github_username": "ceddlyburge",
+    "name": "Cedd Burge",
+    "link_text": "Blog Posts",
+    "link_url": "https://medium.com/@cuddlyburger",
+    "avatar_url": https://github.com/ceddlyburge.png,
```

url need to be enclosed in double quotes `"`.
```suggestion
    "avatar_url": "https://github.com/ceddlyburge.png",
```